libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/NotificationToEmailConverterCommon.java

```diff
@@ -69,7 +72,26 @@ public NotificationToEmailConverterCommon() {
         supportUrl = System.getProperty(PROP_NOTIFICATION_SUPPORT_URL);
 
         emailBaseCSS = readContentFromFile(getClass().getClassLoader(), EMAIL_TEMPLATE_CSS);
+        this.notificationUserAuthority = loadNotificationUserAuthority();
```

Comment:
The PR main change is in this file:
1. Load User Authority
2. On getFullyQualifiedEmailAddresses, if User Authority is loaded get the email for the user.
The other classes were changes as I no longer instantiate NotificationToEmailConverterCommon for each of them - I just inject the same one.